src/Parsers/New/CMakeLists.txt

```diff
@@ -68,8 +68,13 @@ target_compile_options (clickhouse_parsers_new
 
     PUBLIC
         -Wno-extra-semi
-        -Wno-inconsistent-missing-destructor-override
 )
+if (COMPILER_CLANG)
+    target_compile_options (clickhouse_parsers_new
+        PUBLIC
```

Comment:
PUBLIC is an awful mistake that was fixed long time ago.

Reply:
As you can see, this is just a moved/conditional directive from the commands below it.